src/OrbitGl/TimeGraph.h

```diff
@@ -123,11 +123,11 @@ class TimeGraph : public orbit_gl::CaptureViewElement {
   [[nodiscard]] bool IsPartlyVisible(uint64_t min, uint64_t max) const;
   [[nodiscard]] bool IsVisible(VisibilityType vis_type, uint64_t min, uint64_t max) const;
 
-  [[nodiscard]] int GetNumDrawnTextBoxes() const { return num_drawn_text_boxes_; }
   [[nodiscard]] TextRenderer* GetTextRenderer() { return &text_renderer_static_; }
   [[nodiscard]] Batcher& GetBatcher() { return batcher_; }
   [[nodiscard]] std::vector<const orbit_client_data::TimerChain*> GetAllThreadTrackTimerChains()
       const;
+  [[nodiscard]] int GetNumVisiblePrimitives() const;
```

Comment:
Is this `GetNumVisiblePrimitives` actually used in anywhere? If the previous method `GetNumDrawnTextBoxes` always returns 0 as `num_drawn_text_boxes` is never updated, maybe we are not using this method at all. If that is the case, I prefer to directly remove this method.

Reply:
We are using it if we have --devmode enabled. This information appears as part of the layout and I needed for checking the number of boxes is not changing after making optimizations.

Reply:
Thanks for the explanation!